roadmap.md

```diff
@@ -0,0 +1,141 @@
+# Roadmap and Vision
+
+## SIG Release Roadmap for 2021 and beyond
+
+This document contains the SIG Release Roadmap for 2021. The status tracking can
+be found at the bottom. More detailed information can be found on the [SIG
+Release][0] and [Release Engineering][1] project boards.
+
+[0]: https://github.com/orgs/kubernetes/projects/23
+[1]: https://github.com/orgs/kubernetes/projects/30
+
+### Primary Focus
+
+Establish a **consumable**, **introspectable**, and **secure** supply chain for
+Kubernetes. As a supply chain we understand the defining, building and
+publishing of Kubernetes related artifacts.
+
+1. **Consumable**: Improving the usability of artifacts by making their
+   consumption easier. This includes being process independent from external
+   companies like Google.
```

Comment:
I understand the historical context of Google's implementation might be implied here, but this feels odd as a sentence.  Should the process north star be more like "...independent of vendor, employer, individual..."?

I can read this current sentence so many ways.

Is the problem implied here that people can't read from gcr.io because it is Google or can't know what's happening when planning is in Google Docs if access to that is blocked?  Or that the build is depending to Google Cloud software infrastructure?  Or that Google employees have been required for certain steps in the process?  Does Google-independence require shifting hosting to another provider?  Would the same problems not potentially repeat elsewhere without a more abstracted primary focus?

Reply:
Thank you for the input, changed the sentence as suggested. :+1: